### PR TITLE
Remove docker image after push to save space

### PR DESCRIPTION
**Pull request by ivy-cst**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -56,22 +56,26 @@ def buildEdirectory(){
   sh 'wget http://zugpronas:5000/fbsharing/ofPMJOCK -O /tmp/edirectory.tar.gz'
   sh 'tar -xvf /tmp/edirectory.tar.gz -C /tmp'
   sh 'docker load --input /tmp/edir920.tar'
-  def image = docker.image("edirectory:9.2.0")
+  def name = "edirectory:9.2.0"
+  def image = docker.image(name)
   docker.withRegistry('https://docker-registry.ivyteam.io', 'docker-registry.ivyteam.io') {
     if (env.BRANCH_NAME == 'master') {
       image.push()
+      sh "docker image rm ${name}"
```
Comment (alexsuter) [inline Jenkinsfile:64]: you can remove it - even if you are not on master branch

File: Jenkinsfile [hunk 2]
```diff
@@ -56,22 +56,26 @@ def buildEdirectory(){
...
   def tag = directory.replace("/", "-")
+  def name = "axonivy/build-container:${tag}"
   echo "Building container tag $tag in directory $directory"
   dir (directory) {
     docker.withRegistry('', 'docker.io') {
-      def image = docker.build("axonivy/build-container:${tag}")
+      def image = docker.build(name)
       if (env.BRANCH_NAME == 'master') {
         image.push()
+        sh "docker image rm ${name}"
```
Comment (alexsuter) [inline Jenkinsfile:78]: you can remove it - even if you are not on master branch

File: Jenkinsfile [hunk 3]
```diff
@@ -95,10 +99,12 @@ def buildOracleImage(String oracleBinaryUrl, String version, String filename, de
     // -v = Version, -s = Standard Edition
     sh "./buildContainerImage.sh -v $version -s"
 
-    baseImage = docker.image(image)
+    def name = "oracle/database:${version}-se2"
+    baseImage = docker.image(name)
     docker.withRegistry('https://docker-registry.ivyteam.io', 'docker-registry.ivyteam.io') {
       if (env.BRANCH_NAME == 'master') {
         baseImage.push()
+        sh "docker image rm ${name}"
```
Comment (alexsuter) [inline Jenkinsfile:107]: you can remove it - even if you are not on master branch

File: Jenkinsfile [hunk 4]
```diff
@@ -110,10 +116,12 @@ def buildOracleImage(String oracleBinaryUrl, String version, String filename, de
     waitUntilDbIsReady(container)
   }
   
-  def dbImage = docker.build("oracle/database-orapdb:${version}-se2", "oracle/${version}")
+  def name = "oracle/database-orapdb:${version}-se2"
+  def dbImage = docker.build(name, "oracle/${version}")
   docker.withRegistry('https://docker-registry.ivyteam.io', 'docker-registry.ivyteam.io') {
     if (env.BRANCH_NAME == 'master') {
       dbImage.push()
+      sh "docker image rm ${name}"
```
Comment (alexsuter) [inline Jenkinsfile:124]: you can remove it - even if you are not on master branch

**Review by alexsuter — APPROVED** [review]

File: Jenkinsfile (hunk 2 above)
Comment (ivy-cst) [reply Jenkinsfile:78]: I thought then is the whole work lost, is that clever?

Comment (alexsuter) [reply Jenkinsfile:78]: I never connected to a jenkins node and tried out the image there. 
I locally develop the image and build it. Here I only build it to ensure that the image can be built.

Comment (ivy-cst) [reply Jenkinsfile]: Ok

**Pull request merged**